tests/dryrun_test.go

```diff
@@ -64,26 +64,33 @@ var _ = Describe("[sig-compute]Dry-Run requests", decorators.SigCompute, func()
 	})
 
 	Context("VirtualMachineInstances", func() {
-		var vmi *v1.VirtualMachineInstance
+		var (
+			vmi       *v1.VirtualMachineInstance
+			namespace string
+		)
 		resource := "virtualmachineinstances"
 
 		BeforeEach(func() {
-			vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+			vmi = libvmi.NewAlpine(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
```

Comment:
Use `libvmi.WithNamespace(testsuite.GetNamespace(nil))` and keep using `vmi.Namespace` instead of introducing a new var?